README.yaml

```diff
@@ -71,4 +101,4 @@ contributors:
   - name: "Jeremy White"
     github: "dudymas"
   - name: "Benjamin Smith"
-    github: "benbentwo"
\ No newline at end of file
+    github: "benbentwo"
```

Comment:
fyi this block doesnt do anything with the latest build-harness readme

Reply:
This readme isn't generated by the docs folder template